lib/chkname.c

```diff
@@ -79,7 +79,7 @@ bool is_valid_user_name (const char *name)
 	long  maxlen;
 
 	/*
-	 * User names length are limited by the kernel
+	 * User name length should be limited by the kernel
```

Comment:
I would rather remove the comment.  Calling sysconf(3) already tells that it's limited by the kernel, doesn't it?  The comment is just a chance of being confusing.

Reply:
The comment is probably redundant, but let's not spend too much time thinking about it :)